crates/forge_loader/src/manifest.rs

```diff
@@ -120,6 +120,21 @@ pub struct AppInfo<'a> {
     pub id: &'a str,
 }
 
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct ModuleResolve<'a> {
+    pub function: Option<&'a str>
```

Comment:
Does this need to be wrapped in an Option? AFAIK, the function attribute has to be present on all resolvers.